tests/integration/create_cluster/complex/expected-v1alpha2.yaml

```diff
@@ -40,17 +44,29 @@ spec:
   - ::/0
   kubernetesVersion: v1.26.0
   masterPublicName: api.complex.example.com
-  networkCIDR: 172.20.0.0/16
+  networkCIDR: 10.0.0.0/16
   networking:
     cni: {}
   nonMasqueradeCIDR: 100.64.0.0/10
   sshAccess:
   - 1.2.3.4/32
   subnets:
-  - cidr: 172.20.128.0/17
+  - cidr: 10.0.128.0/17
     name: us-test-1a
     type: Public
     zone: us-test-1a
+  - cidr: 10.1.0.0/16
+    name: us-test-1a-1
+    type: Public
+    zone: us-test-1a
+  - cidr: 10.2.0.0/16
+    name: us-test-1a-2
+    type: Public
+    zone: us-test-1a
+  - cidr: 10.3.0.0/16
+    name: us-test-1a-3
+    type: Public
+    zone: us-test-1a
```

Comment:
I'm thinking this test should have a second zone. And we should have a private_complex test.

Reply:
Agreed, done.